pkg/build/controller/policy/serial.go

```diff
@@ -20,7 +20,7 @@ type SerialPolicy struct {
 func (s *SerialPolicy) IsRunnable(build *buildapi.Build) (bool, error) {
 	bcName := buildutil.ConfigNameForBuild(build)
 	if len(bcName) == 0 {
-		return false, NewNoBuildConfigLabelError(build)
+		return true, nil
```

Comment:
yes, that was the intent.. if you're creating one off builds, you don't get a scheduling policy since we have no idea what other builds it should be scheduled against.